examples/07-Train-an-xgboost-model-using-the-Merlin-Models-API.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>I don't think there's anything grammatically wrong with "...to train with by passing..." but the two prepositions side-by-side caught my eye. I think you can remove "with" and not lose meaning. Up to you.</p><p> </p><p>sugg: s/when constructing/when you construct/</p><p> </p><p>Please add a colon after "the following:"</p><p> </p><p>Maybe instead of "...do something else" it could be "...do something better" or "more powerful".</p><p> </p><p>nit: avoid "xgboost" in plain text. Show it as inline code: <code>xgboost</code> when it seems valuable to suggest coding or XGBoost without any styling if you don't need to suggest coding. It similar to using "Merlin Models" in plain text.</p><p> </p><p>sugg: s/we will be setting/we will set/.  Future tense seems OK here.  Also, "Given this piece of information, the Merlin Models code can infer that we want to train..."</p><p> </p><p>sugg: s/would not be/is not/: "...it further, it is not useful for training."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/522/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>